src/viewPanel/gridStackItem.ts

```diff
@@ -108,9 +136,11 @@ export namespace GridStackItem {
     itemTitle?: string;
     pos: number[];
     size: number[];
+    tabName: string;
   }
 
   export interface IChange {
-    action: 'close' | 'lock' | 'edit';
+    action: 'close' | 'lock' | 'edit' | 'layer';
+    dataLayer?: string;
```

Comment:
I'm not really satisfied by this but this was the easiest way to manage the change.
We could find something more generic since other futur changes may require arguments.